server/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const service = require('./controllers.js');
+
+const app = express();
+
+app.use(express.static('public'));
+app.get('/app.bundle.js', (req, res) => {
+  res.sendFile('public/app.bundle.js', { root: '../' });
+});
```

Comment:
This is interesting and I need to look into more on how this works as it's fairly different than mine.